spec/system/task_spec.rb

```diff
@@ -2,23 +2,22 @@
 
 RSpec.describe 'Task', type: :system do
   describe 'Task一覧' do
+    let(:project){ create(:project) }
+    let!(:task){ create(:task, project_id: project.id) }
```

Comment:
taskのFactoryで`association`を定義すると、関連リソースのIDが自動で登録されるようになります。
```suggestion
    let!(:task){ create(:task) }
```

Reply:
376d69b で修正を確認。